### DISCUSSION TITLE
Batocera in French

### DISCUSSION THREAD

**Discussion by lorenzogamingtus** [rg35xx-cfw/rg35xx-cfw.github.io#60] [Q&A] [answered]
Hello, I wanted to know if Batocera maybe in French

**Comment by acmeplus** [comment]
Sorry for the late answer, I clearly missed this thread. Batocera is already translated to many languages, but I believe there's a bug in my fork or it's missing some packages to get it to work. For instance I've tried changing the language settings and the menus are still in English, but the theme sections like Favorites, and Last played, are showing up on the selected language.

Will investigate.

**Comment by lorenzogamingtus** [comment]
cool thank you

**Comment by grandsion** [comment, marked as answer]
merci pour le français dans les menus
